_posts/0000-01-02-teacup69.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text 
```

Comment:
"The pessimist sees difficulty in every opportunity. The optimist sees opportunity in every difficulty." -**Winston Churchill**